images/docs-builder/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -e
+
+if [ $# -lt 1 ]; then
+	echo "You need to provide an argument [build_docs, build_docs_pdf, check_links, docs_serve]"
+fi
+
+case $1 in
+	build_docs)
+		echo "Generating html from docs source"
+		shift 
+		asciidoctor $@
+		;;
+	build_docs_pdf)
+		echo "Generating pdf from docs source"
+		shift 
+		asciidoctor-pdf $@
+		;;
+	check_links)
+		echo "Checking if all links are alive in docs source"
+		find . -name \*.adoc | xargs -n1 asciidoc-link-check -q
+		;;
+	docs_serve)
+		cd build
+		echo "Make sure port 8088 of the container is exposed"
```

Comment:
The `Make sure` confuses me :-/ ` docs can be browsed at http://localhost:8080`?

Reply:
That is already printed by http server :)

```
podman run -it -v /home/anjan/github.com/code-ready/crc/docs:/docs:Z --rm -p 8088:8088/tcp quay.io/anjan/docs-builder docs_serve 
Make sure port 8088 of the container is exposed
Serving HTTP on 0.0.0.0 port 8088 (http://0.0.0.0:8088/) ...
^C
Keyboard interrupt received, exiting.
``` 

so when doing `make serve_docs` this is not really help but if the image is separately used and you just run `podman run -it docs-builder docs_serve` then this will remind you to expose the port. we can remove it..

Reply:
Hmm yeah I think I would just remove it, if someone wants to run this by hand, it's their problem to use the right commandline :)